src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,68 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //ManufacturerService Test
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        System.out.println("Create manufacturers");
+        Manufacturer bmw = new Manufacturer(null, "bmw", "Germany");
```

Comment:
```suggestion
        Manufacturer bmw = new Manufacturer("bmw", "Germany");
```
create one more constructor without id